config/logger.config.js

```diff
@@ -0,0 +1,14 @@
+
+const { format, createLogger, transports} = require('winston');
```

Comment:
`const { format, createLogger, transports }`
or
`const {format, createLogger, transports}`
But not both options at the same line